gavel/templates/admin_annotator.html

```diff
@@ -24,8 +24,8 @@
         <td>{{ annotator.description | safe }}</td>
       </tr>
       <tr>
-        <th>Secret</th>
-        <td>{{ annotator.secret }}</td>
+        <th>Login URL</th>
+        <td><a href="{{ login_link }}">{{ login_link }}</a></td>
```

Comment:
Would it be better to have a relative link here (using just the `url_for()`, as we do below for next and previous items?

If someone right clicks and does "copy link address", they'll still get a full address from their browser. But that way, at least this is not `BASE_URL` dependent (even though emailing has to be because the web app can't figure out its own address).

I'm not sure if this is a good thing or bad thing. Are there any situations where it could cause confusion?